app/src/main/java/org/gdg/frisbee/android/chapter/InfoFragment.java

```diff
@@ -291,11 +334,11 @@ private String getGPlusIdFromPersonUrl(Urls personUrl) {
                 .replace("u0", "")
                 .replace("https:", "")
                 .replace("http:", "")
-                .replace(plusId, "").replaceAll("[^\\d.]", "").substring(0, 21);
+                .replace(chapterPlusId, "").replaceAll("[^\\d.]", "").substring(0, 21);
         }
     }
 
-    private void setIsLoading(boolean isLoading) {
+    void setIsLoading(boolean isLoading) {
```

Comment:
This is removed because IDE suggested me to do it. I think I enabled a flag for this to be shown. Should we do these? What do you think? This will be better performance wise, but it can also make the code un-clean.